charts/prefect-orion/templates/api/deployment.yaml

```diff
@@ -36,7 +36,7 @@ spec:
           imagePullPolicy: {{ .Values.api.image.pullPolicy }}
           command: ["prefect", "orion", "start", "--host", "0.0.0.0", "--log-level", "WARNING"]
           ports:
-          - containerPort: 4200
+          - containerPort: {{ .Values.api.service.port }}
```

Comment:
I dont see the api service port in the values file -- can you add in and perhaps set the default to 4200?

Reply:
Hey, glad to help. 

Isn't it? https://github.com/PrefectHQ/prefect-helm/blob/26dca12e31fae629bb3d4ffc0af1bbdc0cffbb77/charts/prefect-orion/values.yaml#L7